doc/guide/build.hpp

```diff
@@ -6,7 +6,19 @@ This document discusses how to build mlpack from source. These build directions
 will work for any Linux-like shell environment (for example Ubuntu, macOS,
 FreeBSD etc). However, mlpack is in the repositories of many Linux distributions
 and so it may be easier to use the package manager for your system.  For example,
-on Ubuntu, you can install mlpack with the following command:
+on Ubuntu, you can install mlpack library and command-line executables (eg. mlpack_pca, mlpack_kmeans, etc.) with the following command:
+
+@code
+$ sudo apt-get install libmlpack-dev mlpack-bin
+@endcode
+
+On Fedora or Red Hat(EPEL):
+
+@code
+$ sudo dnf install mlpack-devel mlpack-bin
+@endcode
+
+For installing only header files and lib for development purposes one could use:
```

Comment:
```suggestion
For installing only the header files and library for building C++ applications on
top of mlpack, one could use:
```
Just a simple suggestion for clarification.  Otherwise we might imply that one can work on the mlpack source code this way, which is not true.